services/kubernetesbuilddeploy/src/index.ts

```diff
@@ -19,6 +19,9 @@ const lagoonGitSafeBranch = process.env.LAGOON_GIT_SAFE_BRANCH || "master"
 const lagoonVersion = process.env.LAGOON_VERSION
 const overwriteKubectlBuildDeployDindImage = process.env.OVERWRITE_KUBECTL_BUILD_DEPLOY_DIND_IMAGE
 const registry = process.env.REGISTRY || "registry.lagoon.svc:5000"
+const monthlyBackupRetention = process.env.MONTHLY_BACKUP_DEFAULT_RETENTION || "1"
```

Comment:
currently we don't define any monthly backup retention, is that intentional changed to 1?

Reply:
Since we're keeping 6 weekly backups already, this monthly backup should line up with those weekly backups. I can change this to `0` easily enough, though , if you'd prefer?

Reply:
mhh okay, let's leave it at `1` then.